python/dataproc_templates/gcs/README.md

```diff
@@ -63,3 +63,74 @@ export JARS="gs://spark-lib/bigquery/spark-bigquery-latest_2.12.jar"
     --gcs.bigquery.output.mode=<append|overwrite|ignore|errorifexists> \
     --gcs.bigquery.temp.bucket.name="<temp-bq-bucket-name>"
 ```
+
+## GCS To JDBC
+
+Template for reading files from Google Cloud Storage and writing them to a JDBC table. It supports reading JSON, CSV, Parquet and Avro formats.
+
+# Arguments
+
+* `gcs.jdbc.input.format`: Input file format (one of: avro,parquet,csv,json)
+* `gcs.jdbc.input.location`: GCS location of the input files (format: `gs://BUCKET/...`)
+* `gcs.jdbc.output.table`: BigQuery output table name
+* `gcs.jdbc.output.mode`: Output write mode (one of: append,overwrite,ignore,errorifexists)(Defaults to append)
+* `gcs.jdbc.output.driver`: JDBC output driver name
+* `gcs.jdbc.batch.size`: JDBC output batch size
+* `gcs.jdbc.output.url`: JDBC output URL
+
+# Usage
+
+```
+$ python main.py --template GCSTOJDBC --help
+
+usage: main.py --template GCSTOJDBC [-h] \
+    --gcs.jdbc.input.location GCS.JDBC.INPUT.LOCATION \
+    --gcs.jdbc.input.format {avro,parquet,csv,json} \
+    --gcs.jdbc.output.table GCS.JDBC.OUTPUT.TABLE \
+    --gcs.jdbc.output.mode {overwrite,append,ignore,errorifexists} \
+    --gcs.jdbc.output.driver GCS.JDBC.OUTPUT.DRIVER \
+    --gcs.jdbc.batch.size GCS.JDBC.BATCH.SIZE \
+    --gcs.jdbc.output.url GCS.JDBC.OUTPUT.URL
+
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --gcs.jdbc.input.location GCS.JDBC.INPUT.LOCATION
+                        GCS location of the input files
+  --gcs.jdbc.input.format {avro,parquet,csv,json}
+                        Input file format (one of: avro,parquet,csv,json)                        
+  --gcs.jdbc.output.table GCS.JDBC.OUTPUT.TABLE
+                        JDBC output table name
+  --gcs.jdbc.output.mode {overwrite,append,ignore,errorifexists}
+                        Output write mode (one of: append,overwrite,ignore,errorifexists) (Defaults to append)                        
+  --gcs.jdbc.output.driver GCS.JDBC.OUTPUT.DRIVER
+                        JDBC Output Driver Name
+  --gcs.jdbc.batch.size GCS.JDBC.BATCH.SIZE
+                        Batch size of the data means number of records wanted to insert in one round trip into JDBC Table                                               
+  --gcs.jdbc.output.url GCS.JDBC.OUTPUT.URL
+                        JDBC Driver URL to connect with consisting of username and passwprd as well
+```
+
+# Required JAR files
+
+This template requires the JDBC jar file to be available in the Dataproc cluster.
```

Comment:
This is the second situation, because jar file is different to connect different databases. Hence, added and example of wget command for the user with some instructions.

Reply:
Done